src/org/usfirst/frc2175/command/group/RunIntakeOutGroup.java

```diff
@@ -1,26 +1,19 @@
 package org.usfirst.frc2175.command.group;
 
-import org.usfirst.frc2175.command.single.RunDreamIntakeMainBeltAtSpeedCommand;
-import org.usfirst.frc2175.command.single.RunDreamIntakeSideBeltAtSpeedCommand;
+import org.usfirst.frc2175.command.single.RunRollerbarIntakeAtSpeedCommand;
 import org.usfirst.frc2175.config.IntakeConfig;
 import org.usfirst.frc2175.subsystem.RobotSubsystems;
 
 import edu.wpi.first.wpilibj.command.CommandGroup;
 
-/**
- *
- */
 public class RunIntakeOutGroup extends CommandGroup {
 
     public RunIntakeOutGroup(RobotSubsystems robotSubsystems,
             IntakeConfig intakeConfig) {
-        double sideBeltSpeed = intakeConfig.getSideBeltSpeedReverse();
-        double mainBeltSpeed = intakeConfig.getMainBeltSpeedReverse();
+        double rollerbarSpeed = intakeConfig.getRollerbarSpeedOut();
 
-        addParallel(new RunDreamIntakeSideBeltAtSpeedCommand(robotSubsystems,
-                sideBeltSpeed));
-        addSequential(new RunDreamIntakeMainBeltAtSpeedCommand(robotSubsystems,
-                mainBeltSpeed));
+        addSequential(new RunRollerbarIntakeAtSpeedCommand(robotSubsystems,
+                rollerbarSpeed));
 
```

Comment:
Looks like there's some unnecessary whitespace at the bottom of these constructors. You might as well get rid of it.